cassandra-migration/src/main/java/org/cognitor/cassandra/migration/Database.java

```diff
@@ -135,6 +151,29 @@ private void createSchemaTable() {
         session.execute(format(CREATE_MIGRATION_CF, SCHEMA_CF));
     }
 
+    /**
+     * Calculate script checksum and compare it to committed checksum
+     * @param migration
+     */
+    public String validateChecksum(DbMigration migration) {
```

Comment:
I get the point here but I am personally not a big fan of having methods whose result is implicit by what is returned. I think it should either return a ValidationResult or throw an exception if the validation failed instead of having this implicit "if not result is given than everything is fine"